### PR TITLE
Create alireza.txt

### PR DESCRIPTION
**Pull request by AlirezaAkbarix**
desc ...

### CODE REVIEW

**Review by alimd — CHANGES_REQUESTED** [review]
change plz ...

**Review by alimd — APPROVED** [review]
ok

File: alireza.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+hi
```
Comment (alimd) [inline alireza.txt:1]: aleyk hi

Comment (AlirezaAkbarix) [reply alireza.txt:1]: ok

**Comment by developerium** [comment]
manam hi

**Pull request merged**